examples/transfer-hbar2contracts-sdk/transferHbar2Contract_sdk.js

```diff
@@ -0,0 +1,92 @@
+console.clear();
+require("dotenv").config();
+const {
+	AccountId,
+	PrivateKey,
+	Client,
+	AccountCreateTransaction,
+	Hbar,
+	ContractCallQuery,
+	TransferTransaction,
+	ContractInfoQuery,
+	ContractCreateFlow,
+} = require("@hashgraph/sdk");
+const fs = require("fs");
+
+// Configure accounts and client
+const operatorId = AccountId.fromString(process.env.OPERATOR_ID);
+const operatorKey = PrivateKey.fromString(process.env.OPERATOR_PVKEY);
+const aliceKey = PrivateKey.generateED25519();
+const client = Client.forTestnet().setOperator(operatorId, operatorKey);
+
+async function main() {
+	// Create additional accounts needed
+	const initialBalance = 100;
+	const [accStatus, aliceId] = await accountCreatorFcn(aliceKey, initialBalance);
+	console.log(
+		`\n- Created Alice's account with initial balance of ${initialBalance} hbar: ${accStatus}`
+	);
+
+	// Import the compiled contract bytecode
+	const contractBytecode = fs.readFileSync("transferHbar2Contract_sdk_sol_hbar2Contract.bin");
```

Comment:
nit: folder already gives context
```suggestion
	const contractBytecode = fs.readFileSync("hbar2Contract.bin");
```